src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -6,6 +6,7 @@
 public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
+    public static final String MESSAGE_INVALID_VIEW = "This command is invalid here!";
```

Comment:
This is good, we should use this for someone trying to edit from the person list view too